File: Factorial.cpp
//Factorial pf a number.
#include<iostream>
using namespace std;
class Factorial{
	public :
	long int fact(int n){
		if(n>1){
			return n*fact(n-1);
		}
		else
			return 1;
	}
	int factorial(int n){
		int fact=1;
		while(n!=0){
			fact*=n;
			n--;
		}		
		return fact;
	}
};
int main()
{
	Factorial f;
	int n;
	cout<<"Enter the number : ";
	cin>>n;
	cout<<"Factorial using Recursion = "<<f.fact(n);
	cout<<"\nFactorial using loop = "<<f.factorial(n);
	
}

File: Factorial.java
//Write a program that will display the factorial of number 10 using while loop.

class Factorial
{
	public static void main(String args[])
	{
		int n=5,n1=0;
		int fact=1;
		n1=n;
		while(n1!=0)
		{
		 fact=fact*n1;
		 n1--;
		}
		System.out.println("Factorial of "+n+" is "+fact);
	}
}